docs/use-the-network/console/integrations/mqtt.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: mqtt
+hide_title: true
+sidebar_label: MQTT
+slug: /use-the-network/console/integrations/mqtt
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# MQTT
+
+## Add an MQTT Integration
+
+To add an integration, go to **Integrations** on the left-hand menu. Select the
+integration to add - in this case, the **MQTT** integration.
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/console/integrations/integrations-select-mqtt.png"
+  )}
+/>
+
+The next step is to paste your MQTT broker endpoint, and optional topic prefix.
+
+## Connecting Integrations to Devices
+
+Devices are connected to integrations through the use of Labels. Labels are
+named identifiers, that can be used to associate an integration with a device.
+To connect one or more devices to one or more integrations, simply attach the
+same label to both the device and integration. Labels need to be created before
+attaching them to devices and integrations. Read more on this [here](../labels).
+
+## MQTT Topic composition:
+
+### Device Uplink \(Receive Data from Device\)
+
+**Subscribe to:** `{optional_prefix}/helium/{Device ID}/rx`
+
+Example with prefix: `mqtt/helium/3c822699-37fd-4df6-a84d-93037a450843/rx`  
+Example without prefix: `helium/3c822699-37fd-4df6-a84d-93037a450843/rx`
+
+### Device Downlink \(Send Data to Device\)
+
+**Publish to:** `{optional_prefix}/helium/{Device ID}/tx`
+
+Example with prefix: `mqtt/helium/3c822699-37fd-4df6-a84d-93037a450843/tx`  
+Example without prefix: `helium/3c822699-37fd-4df6-a84d-93037a450843/tx`
+
+## MQTT Messages
+
+The MQTT for downlink and uplink follow the
+[JSON schema documented here.](json-schema)
+
+## Connecting Integrations to Devices
+
+Devices are connected to integrations through the use of Labels. Labels are
+named identifiers, that can be used to associate an integration with a device.
+To connect one or more devices to one or more integrations, simply attach the
+same label to both the device and integration. Labels need to be created before
+attaching them to devices and integrations. Instructions for doing this are
+[here](../labels).
+
+## Send and Receive Device Data
+
+The following examples are done with the
+[MQTT CLI](https://hivemq.github.io/mqtt-cli) utility.
+
+Example MQTT
```

Comment:
Not sure this is needed.

Reply:
The link to the CLI tool or examples all together?